pkg/auth/dependency/userverify/sender.go

```diff
@@ -103,6 +104,37 @@ func (t *SMSCodeSender) Send(verifyCode VerifyCode, user response.User) (err err
 	return
 }
 
+type DebugCodeSender struct {
```

Comment:
If we want to debug locally, I suggest we test against a real SMTP server. The Gmail SMTP server is a very common choice when you have an account without 2FA. Let's remove this.